tools/tower/src/core/backlog.rs

```diff
@@ -18,9 +18,14 @@ use libra_types::{
 };
 
 const EPOCH_MINING_THRES_UPPER: u64 = 72;
+
+pub enum BacklogResult {
+    Success,
+    AboveLimit,
+}
 /// Submit a backlog of blocks that may have been mined while network is offline.
 /// Likely not more than 1.
-pub async fn process_backlog(config: &AppCfg) -> anyhow::Result<()> {
+pub async fn process_backlog(config: &AppCfg) -> anyhow::Result<BacklogResult> {
```

Comment:
`tower start` doesn't seem to invoke this method

Reply:
that's intentional